pymc3/distributions/distribution.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
this would go at the top of `distributions/continuous.py` (but see my comment there!)